src/lib/mina_metrics/prometheus_metrics/mina_metrics.ml

```diff
@@ -344,13 +343,20 @@ module Transaction_pool = struct
     in
     Counter.v "transactions_added_to_pool" ~help ~namespace ~subsystem
 
-  let parties_transaction_size : Gauge.t =
+  let zkapp_transaction_size : Gauge.t =
     let help = "Size of valid parties transaction received (bin_size_t)" in
-    Gauge.v "parties_transaction_size" ~help ~namespace ~subsystem
+    Gauge.v "zkapp_transaction_size" ~help ~namespace ~subsystem
```

Comment:
We may want to do a similar total size counter and occurence counter setup for the transaction sizes as well, so that we can also graph this over time.